tests/sentry/workflow_engine/processors/test_detector.py

```diff
@@ -121,9 +121,16 @@ def test_state_results_multi_group(self, mock_produce_occurrence_to_kafka):
         )
 
     def test_no_issue_type(self):
-        detector = self.create_detector(type="invalid slug")
```

Comment:
with this PR we can no longer create a detector with an invalid slug, so substituting with creating a valid detector and mocking that the GroupType gets deleted